lib/Dialect/ESI/capnp/Schema.cpp

```diff
@@ -70,17 +77,23 @@ struct TypeSchemaImpl {
 public:
   TypeSchemaImpl(Type type) : type(type) {}
 
-  /// Get the Cap'nProto schema ID for a type.
+  mlir::Type getType() const { return type; }
```

Comment:
Since I noticed you removed the `mlir::` namespace below, I don't think you need it here either?